### ISSUE TITLE
The aria-label values for the timeline icons are not context specific

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4381]
## Describe the bug

[Test URL]
https://clarity.design/documentation/timeline

[WCAG 2.1 Guideline]
Success Criterion 2.4.6 Headings and Labels
(Level AA)
https://www.w3.org/TR/WCAG21/#headings-and-labels
https://www.w3.org/WAI/WCAG21/Techniques/general/G131

[OS]
any

[User Agents/Assistive Technologies]
any

[Component]
Timeline

[Description]
The aria-label values for the timeline icons are not context specific.

[User Impact]
Screen reader users will hear these phrases - "Not started", "Current", "In progress", "Completed" and "Error" when the read the icons in the timeline. Due to the source order, these icons are read before the textual content that describes them. This means that screen reader users do not get to read the contextual information required to understand the icons before they read them for the first time.

[Steps to Reproduce]
- Go the Test URL
- Using a screen reader, read the icons in timeline and observe that their names lack context

[Recommendation]
Use more contextual aria-label values e.g.,
"Current step in adding KMS", "update for adding kms in progress", "Completed connection to trust center 567" and "Error - could not connect to trust center 566" when the read the icons in the timeline

**Comment by gnomeontherun** [comment]
The static timeline has the ability to customize those labels easily. We can't assume that merging the phrases (not started, current, etc) with the title. I believe we addressed this @hippee-lee in the Angular timeline with offscreen text right? So this should just be minor changes to the static aria labels.

**Comment by hippee-lee** [comment]
Correct, icons get an aria-label description from the one of the Timeline commonStrings. This makes it overridable by the app. 

Common String Name | Default application text
-- | --
timelineStepNotStarted | Used in the aria-label for the not started step icon
timelineStepCurrent | Used in the aria-label for the current step icon
timelineStepSuccess | Used in the aria-label for the success step icon
timelineStepError | Used in the aria-label for the error step icon
timelineStepProcessing | Used in the aria-label for the processing step icon
